### PR TITLE
Add SLO measuring cluster ingress availability using canary probes

### PR DESCRIPTION
**Pull request by glrf**
## Checklist

- [x] PR contains a single logical change (to build a better changelog).
- [x] Update the documentation.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.

### CODE REVIEW

File: docs/modules/ROOT/pages/runbooks/ingress.adoc [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+include::partial$runbooks/contribution_note.adoc[]
+
+[[canary]]
+== Cluster Ingress Canary
+
+=== icon:glasses[] Overview
+
+This SLI measures the percentage of failed probes to an ingress canary application.
+If this SL= results in an alert, it means the cluster ingress controller is unable to handle some of or all of the incoming HTTP requests.
```
Comment (simu) [inline docs/modules/ROOT/pages/runbooks/ingress.adoc:11]:
```suggestion
If this SLI results in an alert, it means the cluster ingress controller is unable to handle some of or all of the incoming HTTP requests.
```

File: docs/modules/ROOT/pages/runbooks/ingress.adoc [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+In this case you probably see a lot of other issues.
+* *Ingress Controller Failure*: The ingress controller might have failed.
+* *Canary Target Failure*: The issue might also just be that the canary targets aren't available.
+In this case this isn't really an issue with the cluster ingress.
+
+Given that the cluster was able to send this alert, a complete outage of the load balancers or the overlay network is unlikely.
+Here are a few first steps you can do to narrow down the issue.
+
+First try to reproduce that you can't reach the canary endpoint yourself.
+Open the canary URL `canary-openshift-ingress-canary.$APP_DOMAIN` from your browser.
```
Comment (simu) [inline docs/modules/ROOT/pages/runbooks/ingress.adoc:85]: Can we add the canary URL to the alert description? Or is it already part of it anyway?

File: docs/modules/ROOT/pages/runbooks/ingress.adoc (hunk 1 above)
Comment (bastjan) [inline docs/modules/ROOT/pages/runbooks/ingress.adoc:11]:
```suggestion
If this SLI results in an alert, it means the cluster ingress controller is unable to handle some of or all of the incoming HTTP requests.
```

**Review by simu — APPROVED** [review]
LGTM overall

File: docs/modules/ROOT/pages/runbooks/ingress.adoc (hunk 2 above)
Comment (glrf) [reply docs/modules/ROOT/pages/runbooks/ingress.adoc:85]: I don't think it is, but that is a good idea.

**Review by bastjan — APPROVED** [review]

**Pull request merged**